docs/components/PlanBuilder.md

```diff
@@ -91,3 +91,10 @@ Additionally, if the connectTo operation is under an interface with name http://
 1. If B has a start or stop operation under a lifecycle interface, before calling the connectTo operation the stop operation on B is called and after the connectTo operation the start operation is called.
 2. If A has a start or stop operation under a lifecycle interface, before calling the connectTo operation the stop operation on B is called and after the connectTo operation the start operation is called.
 
+To circumvent that the connectTo is invoked multiple times on a target node if it connects to another node as a source node use the following naming convention:
+
+1. Whenever the node is the source node: http://opentosca.org/interfaces/connections/source/{type_without_version}
```

Comment:
Use markdown code and define, whether the type is with a double encoded QName or only it's id/name.